scripts/seasonConfigScript.js

```diff
@@ -46,20 +42,19 @@ function addNewSeason()
 //meant for only the startup of the page
 function loadSeasons()
 {
-    if (document.cookie.length > 0)
+    if (getCookie("seasonList").trim().length==0) //if nothing is stored in that cookie yet
     {
-        if (getCookie("seasonList") !== " ")
+        return;
+    }
+
+    var seasonList = getCookie('seasonList').split(SEASON_LIST_SEPARATOR);
+    //console.log('seasonList:' + seasonList);
```

Comment:
don't check in commented out code, use source control to track having it in the past and why you got rid of it.